Source/WebCore/css/CSSFontSelector.h

```diff
@@ -67,7 +68,8 @@ class CSSFontSelector final : public FontSelector, public CSSFontFace::Client, p
     void buildCompleted();
 
     void addFontFaceRule(StyleRuleFontFace&, bool isInitiatingElementInUserAgentShadowTree);
-    void addFontPaletteValuesRule(StyleRuleFontPaletteValues&);
+    void addFontPaletteValuesRule(const StyleRuleFontPaletteValues&);
+    void addFontFeatureValuesRule(const StyleRuleFontFeatureValues&);
```

Comment:
It’s not helpful to include the type names in the names of all these functions. They should just be named `add` or `addRule`. C++ will select the correct function based on the argument type.

Reply:
Same, I'm following the already shipped naming pattern. A renaming only follow-up patch could be a solution though.

Reply:
Same, I'm following the already shipped naming pattern. A renaming only follow-up patch could be a solution though.